src/components/context-menu/ContextMenu.ios.js

```diff
@@ -31,6 +19,12 @@ export default function ContextMenu({
   dynamicOptions,
   onPressActionSheet,
   options = [],
+  destructiveButtonIndex,
+  message,
+  tintColor,
+  title,
+  // eslint-disable-next-line no-unused-vars
+  onPress: _, //to avoid getting onPress in ContextButton
```

Comment:
not sure if this is good. I would choose the existing option tbh.

Reply:
I didn't want it to be confusing that it wasn't used anywhere. Okay, I'll remove it